app/assets/stylesheets/settings.scss

```diff
@@ -330,7 +330,7 @@
       }
       textarea {
         border: 1px solid rgb(222, 230, 233);
-        width: 550px;
+        width: 650px;
```

Comment:
since this is used in multiple places, should we just pull it out into a variable so that 1) it's easier to change in the future and 2) we can reuse it for other forms or inputs that need to be this size should this form change down the road?

Reply:
Updated 👍

Reply:
Yep, that's what I would've done, too :)